contracts/WooRouter.sol

```diff
@@ -186,7 +187,7 @@ contract WooRouter is IWooRouter, Ownable, ReentrancyGuard {
             _generalTransfer(toToken, to, swapBalance);
         }
 
-        emit WooRouterSwap(SwapType.DodoSwap, fromToken, toToken, fromAmount, swapBalance, msg.sender, to);
+        emit WooRouterSwap(SwapType.DodoSwap, fromToken, toToken, fromAmount, swapBalance, msg.sender, to, address(0));
```

Comment:
rebateTo equal to zero address only in externalSwap?

Reply:
yes.  other functions have the parameter "rebateTo"